lib/routes/graphql/schema.gql

```diff
@@ -1,10 +1,10 @@
 type Query {
-  conversations(limit: Int, offset: Int, type: String): [Conversation]
+  conversations(limit: Int, offset: Int, type: String): [Conversation!]
```

Comment:
So conversations can either be 
1. `null` or
2. an array with only non-null members

Do we want it stricter? As in `[Conversation!]!`? Since you mentioned that our resolvers will always give us at least an empty array. That means that getting back `null` is actually illegal.

Reply:
```suggestion
  conversations(limit: Int, offset: Int, type: String): [Conversation!]!
```

Otherwise result could be null